node/pkg/dal/api/controller.go

```diff
@@ -127,15 +138,15 @@ func (c *Controller) handleWebsocket(conn *websocket.Conn) {
 		}
 
 		if msg.Method == "SUBSCRIBE" {
-			_ = db.QueryWithoutResult(
-				context.Background(),
-				"INSERT INTO logs (message) VALUES (@message);",
-				map[string]any{"message": "websocket subscribed(" + strings.Join(msg.Params, ",") + ") from " + conn.IP()},
-			)
+
 			if c.clients[conn] == nil {
 				c.clients[conn] = make(map[string]bool)
 			}
 			for _, param := range msg.Params {
+				err = stats.InsertWebsocketSubscription(ctx, id, param)
+				if err != nil {
+					log.Error().Err(err).Msg("failed to insert websocket subscription")
+				}
 				symbol := strings.TrimPrefix(param, "submission@")
 				if _, ok := c.configs[symbol]; !ok {
 					continue
```

Comment:
how about moving this check to the top of the loop? that way we won't insert invalid websocket subscription. Or is this intentional?

Reply:
agreed